modules/tensor_mechanics/include/materials/ADPowerLawCreepStressUpdate.h

```diff
@@ -41,11 +41,8 @@ class ADPowerLawCreepStressUpdate : public ADRadialReturnCreepStressUpdateBase<c
   virtual ADReal computeDerivative(const ADReal & effective_trial_stress,
                                    const ADReal & scalar) override;
 
-  /// Flag to determine if temperature is supplied by the user
-  const bool _has_temp;
-
   /// Temperature variable value
-  const VariableValue & _temperature;
+  const ADVariableValue & _temperature;
```

Comment:
@dschwen I changed this to AD since power law will certainly be coupled to a temperature primal variable in production runs

Reply:
Moved to #13317